crates/uv/src/main.rs

```diff
@@ -256,12 +256,12 @@ struct PipCompileArgs {
     find_links: Vec<FlatIndexLocation>,
 
     /// Allow package upgrades, ignoring pinned versions in the existing output file.
-    #[clap(long)]
+    #[clap(long, short = 'U')]
     upgrade: bool,
 
     /// Allow upgrades for a specific package, ignoring pinned versions in the existing output
     /// file.
-    #[clap(long)]
+    #[clap(long, short = 'P')]
```

Comment:
I find `-P` unexpected, but I see that this is what pip-tools does.

Reply:
Me too 🤷‍♀️ seems okay for this interface